website-custom-views/src/content/api-reference/commercetools-frontend-application-components.mdx

```diff
@@ -1130,7 +1130,7 @@ const ChannelsInfo = () => {
 | `isOpen`            | `boolean`  |    ✅    | -                 | -       | Indicates whether the dialog is open or closed. The parent component needs to manage this state.                                                                                                                                                                                                                |
 | `onClose`           | `function` |    ✅    | -                 | -       | Called when the dialog closes: click in overlay, click in close button, press `ESC`.                                                                                                                                                                                                                              |
 | `children`          | `node`     |    ✅    | -                 | -       | Content rendered within the dialog. If the content is long in height (depending on the screen size) a scrollbar will appear.                                                                                                                                                                                    |
-| `size`              | `string`   |    -     | `m`, `l`, `scale` | `l`     | Horizontal width of the dialog card. If `scale` is used, the dialog size expands to the viewport size, with some margin.                                                                                                                                                                                                                                                                            |
```

Comment:
I think the comment is still valid regarding `scale`, we don't need to change that.

Reply:
Yes, I believe so, thanks